ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/saveasfragment_dialog/.content.xml

```diff
@@ -0,0 +1,84 @@
+            <name
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                fieldLabel="Name"
+                required="{Boolean}true"
+                name="name">
+                <granite:data
+                        jcr:primaryType="nt:unstructured"
+                        node-name-validation=""/>
+            </name>
+            <title
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                    fieldLabel="Title"
+                    name="jcr:title">
+            </title>
+            <targetPath
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/pathfield"
+                    nodeTypes="nt:folder, sling:Folder, sling:OrderedFolder"
+                    filter="folder"
+                    fieldLabel="Target Path"
+                    name="targetPath"
+                    required="{Boolean}true"
+                    rootPath="/content/dam/formsanddocuments"/>
+            <templateselector
+                    granite:class="cmp-adaptiveform-panelcontainer__templateselector"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete"
+                    emptyText="Type here or browse"
+                    fieldLabel="Select Fragment Template"
+                    filter="template"
+                    name="templatePath"
+                    required="{Boolean}true">
+                <datasource
+                        jcr:primaryType="nt:unstructured"
+                        fdVersion="2"
+                        sling:resourceType="fd/fm/gui/components/admin/formelements/templatedataprovider"/>
+                <options
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete/list"/>
+                <values
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete/tags"/>
+            </templateselector>
+            <fragmentselectcontainer
+                jcr:primaryType="nt:unstructured"
+                granite:class="coral-Form-fieldwrapper cmp-adaptiveform-saveasfragment__fragmentselector-container"
+                sling:resourceType="granite/ui/components/coral/foundation/container"/>
+            <formmodel
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/include"
+                    path="/libs/core/fd/components/form/container/v2/container/cq:dialog/content/items/tabs/items/dataModel/items/formmodel">
```

Comment:
This path should be relative, why is libs mentioned here

Reply:
my bad,  let me correct !!